recipe/meta.yaml

```diff
@@ -1,37 +1,43 @@
-{% set version = "5.5" %}
+{% set version = "6.2" %}
 
 package:
   name: coverage
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/coverage/coverage-{{ version }}.tar.gz
-  sha256: ebe78fe9a0e874362175b02371bdfbee64d8edc42a044253ddf4ee7d3c15212c
+  sha256: e2cad8093172b7d1595b4ad66f24270808658e11acf43a8f95b41276162eb5b8
 
 build:
-  number: 2
+  number: 0
   entry_points:
     - coverage = coverage.cmdline:main
+   skip: True  # [py<36]
```

Comment:
This skip selector does not match when upstream says is the minimum supported Python version.

Reply:
support for `python 2.7 & 3.5` has been dropped since version `6.0` of `coverage` 
https://github.com/nedbat/coveragepy/commit/767a3de326cf438d4731cfddb3e58e6cba3fb5e1